### PR TITLE
Add partition schemes and partition functions to Azure dependencies script

### LINKED ISSUE #129: Table to Partition Scheme and Partition Scheme to Partition Function dependencies are not discovered in Azure
Calling `DiscoverDependencies(table, DependencyType.Parents)` on a `DependencyWalker` object with a table fails to discover the partition scheme & partition function for that table on Azure.

### PR DESCRIPTION
**Pull request by alitas**
Fixes #129. The current script for Azure is missing these queries, causing the discovery process to fail to recognize table to partition scheme and partition scheme to table dependencies.

### CODE REVIEW

File: src/Microsoft/SqlServer/Management/SqlEnum/xml/CloudDependency.sql [hunk 1]
```diff
@@ -480,6 +497,25 @@ begin
 			join sys.tables as tbl on tbl.object_id = fk.referenced_object_id
 			where @iter_no = t.rank and t.object_type = @u and (t.object_svr IS null and t.object_db = db_name())
 		set @rows = @rows + @@rowcount
+                
+                -- Partition Schemes referenced by tables/views
+                insert #t1 (object_id, object_name, object_db, object_type, relative_id, relative_name, relative_schema, relative_db, relative_type, schema_bound, rank)
+                    select ps.data_space_id, ps.name, t.object_db, @part_sch, t.object_id, t.object_name, t.object_schema, t.object_db, t.object_type, 1, @iter_no + 1
+                    from #t1 as t
```
Comment (shueybubbles) [inline src/Microsoft/SqlServer/Management/SqlEnum/xml/CloudDependency.sql:504]: looks like this file uses tabs for SQL indent.
Switch to tabs for now then I can reformat the entire doc to spaces in another change.

**Comment by shueybubbles** [comment]
thx for the PR!
Please open a related Issue to track it.
Also, please include a test that fails without your change.

File: src/FunctionalTest/Smo/ScriptingTests/Table_SmoTestSuite.cs [hunk 2]
```diff
@@ -1566,6 +1566,65 @@ public void VerifyCorrectScriptingOrderWithTemporalTables()
...
+                    }
+
+                    database.Refresh();
+
+                    SqlSmoObject[] tables = database.Tables.Cast<SqlSmoObject>().ToArray();
+                    var dependencyWalker = new DependencyWalker(database.Parent);
+                    var dependencyTree = dependencyWalker.DiscoverDependencies(tables, DependencyType.Parents);
+                    var dependencies = dependencyWalker.WalkDependencies(dependencyTree);
+
+                    Assert.AreEqual(3, dependencies.Count, "Not all dependencies are discovered");
```
Comment (shueybubbles) [inline src/FunctionalTest/Smo/ScriptingTests/Table_SmoTestSuite.cs:1611]: >Assert.AreEqual [](http://example.com/codeflow?start=20&length=15)

pls use the nunit constraints

Assert.That(dependencies.Count, Is.EqualTo(3), "message")

**Comment by shueybubbles** [comment]
git is treating partitionedtable.sql as a binary file instead of a text file

**Review by shueybubbles — APPROVED** [review]
Thx for working on this! We'll cherry pick this into the internal repo for a future release.

**Review by Matteo-T — APPROVED** [review]
:shipit:

**Pull request merged**